frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -118,9 +134,14 @@ export class BitbucketService extends BaseService {
   };
 
   getFileContent = async (path: string): Promise<string | null> => {
-    const url = `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
+    const url = this.isServer
+      ? `${this.baseURL}/projects/${this.metadata.owner}/repos/${this.metadata.repoName}/browse/${path}?at=${this.metadata.defaultBranch}`
+      : `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
     try {
       const data = await this.fetchJson(url);
+      if (this.isServer) {
+        return data.lines.map((l) => l.text).reduce((acc, val) => `${acc + val}\n`, '');
+      }
```

Comment:
When using the /raw/ API we don't need this anymore.

```suggestion
      if (this.isServer) {
        return data.lines.map((l) => l.text).reduce((acc, val) => `${acc + val}\n`, '');
      }
```